postgres-appliance/configure_spilo.py

```diff
@@ -470,6 +479,48 @@ def get_dcs_config(config, placeholders):
     return config
 
 
+def write_log_environment(placeholders):
+    log_env = defaultdict(lambda: '')
+    log_env.update({
+        name: placeholders.get(name, '')
+        for name in [
+            'SCOPE',
+            'LOG_ENV_DIR',
+            'LOG_S3_BUCKET',
+            'LOG_BUCKET_SCOPE_PREFIX',
+            'LOG_BUCKET_SCOPE_SUFFIX',
+            'LOG_TMPDIR',
+            'HOSTNAME',
+            'PGLOG'
+        ]
+    })
+
+    region = placeholders['instance_data']['zone'][:-1]
```

Comment:
It should be possible to configure S3 bucket region from outside, like [here](https://github.com/zalando/spilo/pull/227/files#diff-787073732367780d2de9c4a21cfbec09R30)